spec/unit/provider/ec2_volume/v2_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'spec_helper'
+
+provider_class = Puppet::Type.type(:ec2_volume).provider(:v2)
+
+ENV['AWS_ACCESS_KEY_ID'] = 'redacted'
+ENV['AWS_SECRET_ACCESS_KEY'] = 'redacted'
+ENV['AWS_REGION'] = 'sa-east-1'
```

Comment:
These variables won't need to exist here if https://github.com/puppetlabs/puppetlabs-aws/pull/399 is merged.